experimental/segmenter/src/line.rs

```diff
@@ -173,14 +173,14 @@ impl LineBreakSegmenter {
 }
 
 fn get_linebreak_property_utf32_with_rule(
-    property_table: &LineBreakPropertyTable<'_>,
+    property_table: &RuleBreakPropertyTable<'_>,
     codepoint: u32,
     line_break_rule: LineBreakRule,
     word_break_rule: WordBreakRule,
 ) -> u8 {
     if codepoint < 0x20000 {
         let codepoint = codepoint as usize;
-        let prop = property_table[codepoint / 1024][(codepoint & 0x3ff)];
+        let prop = property_table.0.get(codepoint).unwrap_or(0);
```

Comment:
Suggestion, here and elsewhere: use one of your constants here, like `UNKNOWN`, rather than `0`

Reply:
Done in [51d4463](https://github.com/unicode-org/icu4x/pull/1652/commits/51d4463c6f57bdf8a8319c145dab6aae089eb8a9)